RELEASES.md

```diff
@@ -370,5 +370,17 @@ the total stake. The remaining 33% of the stake is configured to belong to
 a validator that is never actually run in the test network. The network is run
 for multiple days, ensuring that it is able to produce blocks without issue.
 
+#### Vote-extension Testnet
+
+CometBFT v0.38.0 introduced **vote-extensions**, which are added as the name suggests, to votes sent by validators.
+The Vote-extension Testnet is used to determine how vote-extensions affect the performance of CometBFT, under various settings.
+The testnet consists of 175 validators, 20 non-validator full-nodes, and 5 seed nodes.
+All 195 full-nodes begin by dialing a subset of the seed nodes to discover peers.
+Once all full-nodes are started, a 5 minute period is waited while load-generators are started and before any load is introduced in the system.
+Once the startup period is over, the load generators issue requests at a constant rate during 150 seconds.
+The load generation step is repeated 5 times with intervals of 60 seconds in between to allow the system to quiesce.
+A wide set of metrics is captured during the whole experiment but, unless abnormalities are observed, the first and last 30 seconds of each 150 interval are discarded; the remainder 90 seconds are used in computing aggregated metrics.
+The application used in the experiment is the same used on the (#200-node-testnet), so the results are comparable.
+
```

Comment:
Should we describe how the length of vote extensions is going to vary during the test execution?

Reply:
Updated the text. Pls have a look.